frontend/src/components/PopularCard/PopularCard.css

```diff
@@ -0,0 +1,89 @@
+a {
+  text-decoration: none;
+}
+
+.card-popular {
+  font-family: 'Roboto', sans-serif;
+  background: white;
+  height: 490px;
+  width: 337px;
+  /*Added border for reference only to see component in Storybook well*/
+  border: 1px solid rgba(61, 94, 15, 0.15);
+}
+
+.course-image {
+  height: 257px;
+}
+
+.course-info {
+  margin-top: -4px;
+  padding: 10px;
+  height: 213px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.course-title {
+  margin: 0;
+  color: #547818;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 25px;
+  line-height: 29px;
+}
+
+.concepts--title {
+  margin: 0;
+  color: #8E9D79;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 20px;
+}
+
+.concepts--list {
+  margin: 0;
+  padding-top: 8px;
+  color: black;
+  font-size: 22px;
+  line-height: 26px;
+}
+
+.course-stats {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.course-stats a {
+  font-weight: bold;
+}
+
+.course-stats a:after {
+  content: "";
+  width: 0;
+  height: 1px;
+  display: block;
+  background: #547818;
+  transition: 300ms;
+}
+
+.course-stats a:hover:after {
+  width: 100%;
+}
+
+.course-stat {
+  font-size: 14px;
+  color: #787575;
+}
+
+.course-stat-br {
+  height: 16px;
+  width: 3px;
+  background-color: #C4C4C4;
+}
+
+.course-apply {
+  font-size: 14px;
+  color: black;
+}
```

Comment:
Add newline at the end...

Reply:
not sure what you mean exactly, just a new empty line at the end of the file? the same question applies to comment below

Reply:
Yes, just a code styling preference. As you prefer...

Reply:
This should be handled by Prettier though

Reply:
Might be that back then.. as I've coded this long ago, I didn't yet have prettier set up somehow, I remember that since WebStorm 2020, they baked it into the IDE, happy to run it through prettier if you like, let me know.

Reply:
For now just adding the line (you can even edit it here in GitHub! no need to go to your editor) is fine, and just keep it in mind for next time!